### PR TITLE
Bmg

### PR DESCRIPTION
**Pull request by bmgdevelopment**
Adjusted Resource main page and now structuring subpgs

### CODE REVIEW

File: pages/resources/couseling.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export default function couseling() {
```
Comment (drteresavasquez) [inline pages/resources/couseling.js]: Probably will update when complete, but make sure the component names are capitalized

File: pages/resources/exercise.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export default function exercise() {
```
Comment (drteresavasquez) [inline pages/resources/exercise.js]: Same here, capitalize

File: pages/resources/help.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from 'react';
+import { supabase } from '../../utils/client';
+import { useRouter } from 'next/router';
+import Loader from '../../components/Loader';
+import MainLayout from '../../layouts/MainLayout';
+import {help} from '../../data/resources'
+
+
+export default function Creative() {
```
Comment (drteresavasquez) [inline pages/resources/help.js]: This needs to be renamed to `Help`

**Review by drteresavasquez — CHANGES_REQUESTED** [review]
A couple of small changes

File: pages/resources/couseling.js (hunk 1 above)
Comment (bmgdevelopment) [reply pages/resources/couseling.js]: Updated!

File: pages/resources/exercise.js (hunk 2 above)
Comment (bmgdevelopment) [reply pages/resources/exercise.js]: Updated!

File: pages/resources/help.js (hunk 3 above)
Comment (bmgdevelopment) [reply pages/resources/help.js]: Updated!

**Review by drteresavasquez — APPROVED** [review]
Not sure about the yarn.lock changes, so we will see if they impact any other team members

**Pull request merged**